packages/origin-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@energyweb/origin-backend",
-    "version": "11.2.1",
+    "version": "11.2.2",
```

Comment:
why do you want to bump that version?

Reply:
misunderstanding. I wil revert

Reply:
reverted on commit [8cf045a](https://github.com/energywebfoundation/origin/pull/3367/commits/8cf045a7655a174211f38dd1f121efe8d0f7c0bc)